### ISSUE TITLE
ValueError: could not convert string to float:

### ISSUE THREAD

**Issue by Hellchampion** [hjelev/rpi-mqtt-monitor#127]
I get this error:

```
pi@Homebridge:~ $ rpi-mqtt-monitor -d
Traceback (most recent call last):
  File "/home/pi/rpi-mqtt-monitor/src/rpi-cpu2mqtt.py", line 780, in <module>
    gather_and_send_info()
  File "/home/pi/rpi-mqtt-monitor/src/rpi-cpu2mqtt.py", line 673, in gather_and_send_info
    cpu_load, cpu_temp, used_space, voltage, sys_clock_speed, swap, memory, uptime_days, uptime_seconds, wifi_signal, wifi_signal_dbm, rpi5_fan_speed = collect_monitored_values()
                                                                                                                                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pi/rpi-mqtt-monitor/src/rpi-cpu2mqtt.py", line 656, in collect_monitored_values
    memory = check_memory()
             ^^^^^^^^^^^^^^
  File "/home/pi/rpi-mqtt-monitor/src/rpi-cpu2mqtt.py", line 90, in check_memory
    memory = round(float(memory.decode("utf-8").replace(",", ".")))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: ''
```
System Info:

PRETTY_NAME="Debian GNU/Linux 12 (bookworm)"
NAME="Debian GNU/Linux"
VERSION_ID="12"
VERSION="12 (bookworm)"
VERSION_CODENAME=bookworm
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"

 Operating System: Debian GNU/Linux 12 (bookworm)
 Kernel: Linux 6.6.20+rpt-rpi-v8
 Architecture: arm64

Linux Homebridge 6.6.20+rpt-rpi-v8 #1 SMP PREEMPT Debian 1:6.6.20-1+rpt1 (2024-03-07) aarch64 GNU/Linux

May 24 2024 15:30:04
Copyright (c) 2012 Broadcom
version 4942b7633c0ff1af1ee95a51a33b56a9dae47529 (clean) (release) (start)

**Comment by pationair** [comment]
Hello, I have the same problem with a fresh installation on a Raspberry 3. However, I use Raspian 11 Bullseye as the operating system

**Comment by diveschumi** [comment]
Same here

**Comment by mdak92** [comment]
I fixed by changing Line 90 in src/rpi-cpu2mqtt.py to

`if not memory:`

(Sorry for not making a PR, I never did this and do not want to mess around.)

**Comment by Hellchampion** [comment]
Thx @mdak92 this will works for me.

**Comment by hjelev** [comment]
I have fixed this, if you reinstall or do a git pull update it should be fixed.
Thanks for reporting it.

**Comment by hjelev** [comment]
The fix for this is added to 0.9.4
